config/backstop/backstopConfig.js

```diff
@@ -35,7 +35,36 @@ const config = {
     }
   ],
   scenarios: [
-    { ...basicScenario }
+    {
+      ...basicScenario,
+      label: 'entire-document',
+      selectors: ['document']
+    },
+    {
+      ...basicScenario,
+      label: 'h1-heading',
+      selectors: ['h1']
```

Comment:
are these selectors used to generate screenshots? 

I'd say this PR has too many screenshots. Let's reduce that to 2-3 screenshots?

Reply:
@roman-mate This is a specific task where we need to check how they write `HTML`. Basically, there are some `<p>` tags with `<br>` inside. We also have to make sure they use `h2` for level-2 headings.
Also splitting the entire document to small screenshots helps us to check even `1%` mismatch. 

If it is ok for you to remove `h2` and `p` from checking I can do it as well. This way we'll have 
  - one `document` screenshot  
  - one `h1` screenshot
  - one `a` screenshot

Reply:
yeah, that makes sense. Let's just write a selector that returns one element instead of multiple.

Reply:
for the moment we can't force them to write specific selectors. They have just empty `index.html` and they know nothing about `id`, `class`, `data-attributes` etc. So the best way is to keep only selectors which return a small number of screenshots (like `h1`)

Reply:
How about h1:nth-of-type(100n+1)?

Reply:
good idea, thank you